streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/collector/OutputCollector.java

```diff
@@ -14,26 +19,59 @@
 public class OutputCollector implements Collector<Record> {
   private static final Logger LOGGER = LoggerFactory.getLogger(OutputCollector.class);
 
-  private Partition partition;
   private DataWriter writer;
   private ChannelID[] outputQueues;
+  private Collection<RayActor> targetActors;
+  private Language[] targetLanguages;
```

Comment:
It would be good for further developing that we mark as `final` as possible.

Reply:
Yes, we should do this